lib/src/mapbox_maps_options.dart

```diff
@@ -4,7 +4,9 @@ part of mapbox_maps_flutter;
 final class MapboxOptions {
   static _MapboxOptions _options = _MapboxOptions();
 
-  MapboxOptions._() {}
+  MapboxOptions._() {
+    LogConfiguration._setupDebugLoggingIfNeeded();
```

Comment:
is this ever called?

Reply:
Ah, good point, it is never called indeed, I mistook it for `_MapboxOptions` constructor somehow, changed it to static variable init.